src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -276,6 +276,9 @@
   <data name="net_http_invalid_proxy" xml:space="preserve">
     <value>When using WindowsProxyUsePolicy.UseCustomProxy, the Proxy property must not be null.</value>
   </data>
+  <data name="net_http_handler_nocontentlength" xml:space="preserve">
+    <value>The content length of the request content can't be determined. Either set TransferEncodingChunked to true, load content into buffer, or set AllowRequestContentBuffering to true.</value>
+  </data>
```

Comment:
For this added string, yes.  But we don't usually have IFDEF stuff (don't think it even works) in the *.resx file. For example, unix only strings are in here as well.  The cost is pretty small so we just live with all the strings in the one *.resx file.